plugins/Cardinal/src/HostTime.cpp

```diff
@@ -165,11 +175,25 @@ struct HostTime : TerminalModule {
         outputs[kHostTimeBeat].setVoltage(hasBeat ? 10.0f : 0.0f);
         outputs[kHostTimeClock].setVoltage(hasClock ? 10.0f : 0.0f);
         outputs[kHostTimeBarPhase].setVoltage(barPhase * 10.0f);
-        outputs[kHostTimeBeatPhase].setVoltage(beatPhase * 10.0f);
+        outputs[kHostTimeBeatPhase].setVoltage((tick / pcontext->ticksPerBeat) * 10.0f);
```

Comment:
this seems like a leftover... please restore the original, otherwise for hosts that dont provide time info this results in a division by zero and crash

Reply:
Good catch, have fixed it.